spikeinterface/sortingcomponents/tests/test_waveforms/conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+import spikeinterface as si
+import spikeinterface.extractors as se
+
+from spikeinterface.sortingcomponents.peak_detection import detect_peaks
+
+
+@pytest.fixture(scope="package")
+def chunk_executor_kwargs():
+    job_kwargs = dict(n_jobs=-1, chunk_size=10000, progress_bar=False)
```

Comment:
n_jobs=-1 for tests is maybe a bit agressive.
I like to have n_job=1 and if we really want mltiprocessing n_job=2 or 4.

Reply:
Why not used all the machine resources during testing? Specially for CI where the runners are dedicated.